sdk/tables/tests.yml

```diff
@@ -1,13 +1,6 @@
 trigger: none
 
-resources:
-  repositories:
-    - repository: azure-sdk-tools
-      type: github
-      name: Azure/azure-sdk-tools
-      endpoint: azure
-
-jobs:
-- template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+extends:
+  template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
```

Comment:
nit: but it might be worth using absolute paths for these templates now.

Reply:
I'll see if that works.